cmake/SSGCommon.cmake

```diff
@@ -923,27 +923,28 @@ macro(ssg_build_html_cce_table PRODUCT)
 endmacro()
 
 macro(ssg_build_html_srgmap_tables PRODUCT DISA_SRG_TYPE DISA_SRG_VERSION)
+    FILE(GLOB DISA_SRG_REF "${SSG_SHARED_REFS}/disa-${DISA_SRG_TYPE}-srg-${DISA_SRG_VERSION}.xml")
```

Comment:
cmake style, file should be lowercase
since the regex is the same for all products, shouldn't we just set it here? why set it in the CMakeLists.txt in each product to exactly the same thing?

Reply:
While it isn't the case today, I was planning for if there was a need in case you have `disa-os-srg-v1r4.xml` and `disa-os-srg-v2r10.xml` where one product had to have a specific version that needed to be tied to it. Or should we just not care about that?

Reply:
If that is the case couldn't we just remove one of the files and keep only the one that should be tied to the product?

Reply:
@mpreisler what if the other product(s) require the new versions?

Reply:
every product has their own reference doc in CMAKE_CURRENT_SOURCE_DIR, right? so they can all use different versions, that's fine. As long as every product is using just one version.